lib/token-autocomplete.ts

```diff
@@ -580,7 +580,7 @@ class TokenAutocomplete {
             let hiddenOption = me.parent.hiddenSelect.querySelector('option[data-text="' + tokenText + '"]') as HTMLElement;
             if (!me.options.optional) {
                 me.previousValue = hiddenOption?.dataset.value;
-                me.previousText = hiddenOption?.dataset.tokenText;
+                me.previousText = hiddenOption?.dataset.text;
```

Comment:
So the `hiddenOption` has no `tokenText`? Or why did you revert this one?

Reply:
yeah the hiddenOption has text, value, type ( they no longer need the completion label or completion description )
the suggestion has text, value, type and tokenText, where the tokenText is mapped to text when creating a token/hiddenoption from a suggestion